gui/launcher.cpp

```diff
@@ -255,6 +268,15 @@ void LauncherDialog::build() {
 	// I18N: Button caption. O is the shortcut, Ctrl+O, put it in parens for non-latin (~O~)
 	new ButtonWidget(this, _title + ".OptionsButton", _("Global ~O~ptions..."), _("Change global ScummVM options"), kOptionsCmd, 0, _c("Global ~O~pts...", "lowres"));
 
+#if defined(USE_SCUMMVMDLC) && defined(USE_LIBCURL)
+	// I18N: Button browse downloadable games (DLC)
+	new ButtonWidget(this, _title + ".DownloadGamesButton", _("Download Games"), _("Download freeware games for ScummVM"), kDownloadGameCmd);
+#elif defined(USE_DLC)
```

Comment:
I do not like this code duplication. You could simplify the thing that when `USE_SCUMMVMDLC` is defined, `g_system->hasFeature(OSystem::kFeatureDLC)` is returning `true` and `getDLC()` returns `ScummVMDLC`